app/presenters/publishing_api/organisation_presenter.rb

```diff
@@ -495,5 +497,13 @@ def high_profile_groups_links
     def roles_links
       item.roles.distinct.pluck(:content_id)
     end
+
+    def default_news_image
+      return unless item.default_news_image
+      {
+        url: item.default_news_image.file_url,
+        high_resolution_url: item.default_news_image.file.url(:s960),
```

Comment:
Do we need to wrap these in image_url helpers as per: https://github.com/alphagov/whitehall/blob/1311c133ee97c51e06c62a67139c68454222876d/app/presenters/publishing_api/news_article_presenter.rb#L129 ?

Reply:
I compared the output of the helpers and the file urls and they were exactly the same, so I felt the helpers were a bit redundant.

Reply:
Ah really, is it legacy since the asset manager change? 

Feel bold enough to kill it off in the NewsArticlePresenter?

Reply:
I'll test this out on integration; if it's all ok then I'll open a separate PR to sort out NewsArticlePresenter :)

Reply:
Yup separate sounds better